docs/source/functional.rst

```diff
@@ -26,6 +26,11 @@ create_fb_matrix
 
 .. autofunction:: create_fb_matrix
 
+linear_fbanks
+----------------
```

Comment:
```suggestion
linear_fbanks
-------------
```

Reply:
fc3c1ad. I think the suggested change would have added an additional `linear_fbank`.